webview-ui/src/components/chat/TaskHeader.tsx

```diff
@@ -272,6 +274,13 @@ const TaskHeader: React.FC<TaskHeaderProps> = ({
 								{!isCostAvailable && <ExportButton />}
 							</div>
 
+							<div style={{ display: "flex", alignItems: "center", gap: "4px", flexWrap: "wrap" }}>
+								<span style={{ fontWeight: "bold" }}>Context:</span>
+								<span style={{ display: "flex", alignItems: "center", gap: "3px" }}>
+									{formatLargeNumber(contextTokens || 0)}
```

Comment:
Only nitpick is that I'm not sure 0 is the right fallback, for instance when loading old conversations. Maybe could be something like
```suggestion
									{contextTokens ? formatLargeNumber(contextTokens) : '-'}
```